CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.0.0] - 2020-05-07
+
+## Changed
+
+- Poll GitHub for phishing config file (#232)
+- Rebrand as `@metamask/controllers` (#226)
+- Use yarn & drop `npm-shrinkwrap.json` (#193)
+
+## Removed
+
+- Remove Husky (#234)
```

Comment:
This doesn't affect consumers of this package, so I'm not sure it's worth mentioning.